var/spack/repos/builtin/packages/cp2k/fj_2023.1.patch

```diff
@@ -0,0 +1,32 @@
+diff -Nur cp2k-2023.1.org/src/fm/cp_fm_elpa.F cp2k-2023.1.OK/src/fm/cp_fm_elpa.F
+--- cp2k-2023.1.org/src/fm/cp_fm_elpa.F	2023-07-20 17:03:27.000000000 +0900
++++ cp2k-2023.1/src/fm/cp_fm_elpa.F	2023-09-20 09:00:41.000000000 +0900
+@@ -189,7 +189,9 @@
```

Comment:
This patch is never used in the recipe...

Reply:
@alalazo  My apologies for the oversight. I've added the necessary patch application.